app/registries/foreman/settings/auth.rb

```diff
@@ -131,12 +131,8 @@
     setting('bmc_credentials_accessible',
       type: :boolean,
       description: N_("Permits access to BMC interface passwords through ENC YAML output and in templates"),
-      default: true,
+      default: false,
       full_name: N_('BMC credentials access'))
-    validates('bmc_credentials_accessible',
-      ->(value) { value || Setting[:safemode_render] },
-      message: N_("Unable to disable bmc_credentials_accessible when safemode_render is disabled"))
```

Comment:
This removal doesn't make much sense unless we remove the validation from `safemode_render` as well IMHO.
Also this allows entering the invalid state if users didn't change this setting, but had changed the `safemode_render`